File: frontend/src/types/roomType.ts
export interface RoomType {
  id: number;
  hotelId: number;
  name: string;
  description: string;
  capacity: number;
  dailyRate: number;
}

export interface RoomTypeRequest {
  name: string;
  description: string;
  capacity: number;
  dailyRate: number;
}

File: backend/HotelBookingEngine.Api/Features/RoomTypes/RoomType.cs
namespace HotelBookingEngine.Api.Features.RoomTypes;

public class RoomType
{
    public int Id { get; set; }
    public int HotelId { get; set; }
    public required string Name { get; set; }
    public required string Description { get; set; }
    public int Capacity { get; set; }
    public decimal DailyRate { get; set; }
}
